### ISSUE TITLE
[DatePicker]- Disabled state of up/down arrow buttons can not be visually identified in High Contrast mode

### ISSUE THREAD

**Issue by Bhavana-Nimma** [microsoft/fluentui#18686]
### Environment Information

- **Package version(s)**: 8.20.2
- **Browser and OS versions**:  Microsoft Edge, Windows 10

### Describe the issue:

In High Contrast Black mode and in High Contrast White mode, the disabled state of  up/down arrow buttons i.e., previous/next arrow buttons can not be visually identified

#### Actual behavior:

The below images show the same datepicker in all the three modes. Here we can observe that the disabled arrow button is visually not varied when compared with enabled arrow button in HC modes.

Normal Mode : 

![image](https://user-images.githubusercontent.com/85894656/123136357-7805be00-d470-11eb-83aa-375967518378.png)

HCB mode : 

![image](https://user-images.githubusercontent.com/85894656/123136580-b8fdd280-d470-11eb-81c5-fbf2e1357103.png)

HCW mode : 

![image](https://user-images.githubusercontent.com/85894656/123136892-1003a780-d471-11eb-984b-50b442515c88.png)

#### Expected behavior:

The disabled state of next/previous arrow buttons should follow the similar visual pattern of the other disabled buttons in HC mode

### Please provide a reproduction of the issue in a codepen:

Here is the reproduction of the issue in codepen (the issue can be reproduced in HC mode) - https://codepen.io/bhavana-nimma/pen/bGqPvNy

**Comment by gouttierre** [comment]
@Bhavana-Nimma - thank you for reaching out to us and providing the details around the issue with HC.

@ling1726 -  Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by ling1726** [comment]
This is definitely an a11y bug that should be fixed. The v8 `IconButton` adds a background color when disabled in HC:
![image](https://user-images.githubusercontent.com/20744592/125306938-ec28d880-e32f-11eb-8911-8c65147e4510.png)
![image](https://user-images.githubusercontent.com/20744592/125307064-0498f300-e330-11eb-8c36-460322281ff7.png)
 

Ideally the fix here should follow a similar pattern.

**Comment by smhigley** [comment]
Ah, this is because we use the `aria-disabled` attribute instead of the HTML `disabled` attribute to prevent focus being lost when the buttons become disabled.

That's necessary, but also means we need to add custom high contrast disabled styles :/. We should be able to use the system color keyword for disabled text and set the button text color to that in a forced-colors media query.

**Comment by anowduru** [comment]
@gouttierre Can you let us know the ETA for the fix?

**Comment by ddas-ms** [comment]
@gouttierre can you please help us with the fix ASAP? Like @ling1726 mentioned, "This is definitely an a11y bug that should be fixed". This has become a major blocker for us, and we are already out of SLA. I think, @smhigley proposed something close to a solution.
